mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir

```diff
@@ -1719,6 +1719,40 @@ func.func @gather_with_zero_mask_scalable(%arg0: memref<?xf32>, %arg1: vector<2x
 
 // -----
 
+//===----------------------------------------------------------------------===//
+// vector.scatter
+//===----------------------------------------------------------------------===//
```

Comment:
Do these qualify as "complex tests"L
* https://github.com/llvm/llvm-project/blob/d928a671b84afb9c2ad64353694537a198f04651/mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir#L3-L13

If not, move somewhere here https://github.com/llvm/llvm-project/blob/d928a671b84afb9c2ad64353694537a198f04651/mlir/test/Conversion/VectorToLLVM/vector-to-llvm-interface.mlir#L2152-L2154

Also, given that these are "negative" tests, we could label them accordingly:
* https://mlir.llvm.org/getting_started/TestingGuide/#step-3-add-the-newly-identified-missing-case

Reply:
> Do these qualify as "complex tests"

Yes they do. vector-to-llvm-interface tests do not include any preprocessing. We need preprocessing to unroll multi dimensional scatters. This is exactly same as gather, we are being consistent.

> Also, given that these are "negative" tests, we could label them accordingly:

Sure I will change it. They are anyway going to become positive test in the next pr in the stack: https://github.com/llvm/llvm-project/pull/132227

Reply:
> They are anyway going to become positive test in the next pr in the stack: https://github.com/llvm/llvm-project/pull/132227

Ah, ok. Then you can skip it.

Reply:
Thanks :)